src/requireUniversalModule.js

```diff
@@ -138,16 +138,19 @@ export default function requireUniversalModule<Props: Props>(
       if (chunkName) {
         const name = callForString(chunkName, props)
         if (name) CHUNK_NAMES.add(name)
-        if (!isTest) return // makes tests way smaller to run both kinds
+        if (!isTest) return name // makes tests way smaller to run both kinds
       }
 
       if (isWebpack()) {
         const weakId = callForString(resolve, props)
         if (weakId) MODULE_IDS.add(weakId)
+        return weakId
```

Comment:
im not sure if we would need to return the weakIds for anything...  but it seemed to match the pattern of returning the name above

Reply:
yea, that's fine. that's for the old approach. the only problem is the component is called `<ReportChunks>`, but it doesn't matter. I'm fine iwth it.